packages/primitives-core/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "primitives-core",
```

Comment:
This should be `@emotion/primitives-core` and it should be imported as `@emotion/primitives-core`